app/main/views.py

```diff
@@ -10,6 +10,11 @@ def index():
     return render_template('main/index.html')
 
 
+@main.route('/map')
+def user_map():
```

Comment:
we need an `@login_required` on this route, please make sure this change get's made